src/tooltip.js

```diff
@@ -217,6 +218,18 @@ angular.module('ts.utils')
           }
         }
         else{
+          angular.element($window).on('resize', function(){
+            positionTooltip();
```

Comment:
It needs to be unbound on destroy. For this, the function will need to be somewhere else. You can also use positionTooltip directly

`angular.element($window).on('resize', positionTooltip);`

and then destroy

`angular.element($window).off('resize', positionTooltip);`

you could also do 

`$window.addEventListener('resize', positionTooltip);`
Destroy
`$window.removeEventListener('resize', positionTooltip);`